src/Courant/Courant_presets.jl

```diff
@@ -0,0 +1,54 @@
+module Courant
+
+
+export Advective_CFL, Diffusive_CFL
+
+using LinearAlgebra, StaticArrays
+using ..VariableTemplates
+using ..MoistThermodynamics
+using ..PlanetParameters
+import ..MoistThermodynamics: internal_energy
+using ..SubgridScaleParameters
+using GPUifyLoops
+using ..MPIStateArrays: MPIStateArray
+using CLIMA.Atmos: AtmosModel, soundspeed
+import CLIMA.DGmethods: BalanceLaw, vars_aux, vars_state,
+                        vars_diffusive
+using ..Mesh.Topologies
+using ..Mesh.Grids
+using ..Mesh.Grids: VerticalDirection, HorizontalDirection, EveryDirection
+using Logging
+using Printf
+using LinearAlgebra
+using CLIMA.Atmos
+
+
+
+function Advective_CFL(m::AtmosModel, state::Vars, aux::Vars,
+                               diffusive::Vars, Δx, Δt, direction=VerticalDirection())
+  k̂ = aux.orientation.∇Φ / norm(aux.orientation.∇Φ)
+  if direction isa VerticalDirection
+    u = dot(state.ρu/state.ρ, k̂) * k̂
+          return Δt * (norm(u) + soundspeed(m.moisture, m.orientation, state,aux)) / Δx
+  elseif direction isa HorizontalDirection
+    u = cross(k̂, cross(state.ρu/state.ρ,k̂))      
+    return Δt * (norm(u) + soundspeed(m.moisture, m.orientation, state,
+                                            aux)) / Δx
+  end
+end
```

Comment:
This is very clean and looks good. 

Can we have a third function just to calculate the advective (i.e. without soundspeed) courant?

Reply:
Yes I'll add that in shortly